### PR TITLE
qt5-quick3d: Disable building with system-assimp

### PR DESCRIPTION
**Pull request by MehdiChinoune**
qt5-quick3d does not support the latest assimp 5.1.0

### CODE REVIEW

**Comment by Biswa96** [comment]
Can qt6-quick3d be built with latest assimp? If yes then KDE devs should be notified to import the upstream patches to qt5.

File: mingw-w64-qt5-pkgs/PKGBUILD [hunk 1]
```diff
@@ -252,6 +252,7 @@ build() {
     -system-sqlite \
     -system-zlib \
     -no-iconv \
+    -qt-assimp \
```
Comment (lazka) [inline mingw-w64-qt5-pkgs/PKGBUILD:255]: typo?

**Comment by MehdiChinoune** [comment]
> Can qt6-quick3d be built with latest assimp? If yes then KDE devs should be notified to import the upstream patches to qt5.

No, It can't. I have tried to to build it yesterday, it failed with the same error messages.

**Comment by MehdiChinoune** [comment]
I don't know why `configure.bat` doesn't accept options for other than qtbase?

**Pull request closed without merging**